.github/workflows/python-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Python Tests
+
+on: [push, pull_request]
```

Comment:
Can you also please enable the tests before merge?
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions